packages/downloads/src/portal/portal-poll-export-job-status.ts

```diff
@@ -98,11 +97,11 @@ class PortalPoller implements IPoller {
         .catch((error: any) => {
           if (error instanceof ExportCompletionError) {
             eventEmitter.emit(`${downloadId}ExportError`, {
-              detail: { metadata: { errors: [error] } }
```

Comment:
this makes it look like a breaking change. in order to avoid that we should probably keep `metadata.errors` (in addition to `detail.error`) until the next major release. You can add it to this list: https://github.com/Esri/hub.js/issues/305

Reply:
I've made the change to keep `metadata.errors`.  As of this writing, the code snip above isn't updating to show it, but you can view it [here](https://github.com/Esri/hub.js/blob/p/download-errors/packages/downloads/src/portal/portal-poll-export-job-status.ts#L93-L111).